tests/restarting/from_7.0.0/UpgradeAndBackupRestore-1.toml

```diff
@@ -0,0 +1,54 @@
+[[test]]
+testTitle = 'SubmitBackup'
+simBackupAgents= 'BackupToFile'
+clearAfterTest = false
+
+    [[test.workload]]
+    testName = 'SubmitBackup'
+    delayFor = 0
+    stopWhenDone = false
+
+[[test]]
+testTitle = 'FirstCycleTest'
+clearAfterTest=false
+runConsistencyCheck = false
```

Comment:
Has correctness been run on this yet? This scenario has been known to fail in the first step (test block before this), so `runConsistencyCheck = false` may need to be moved there instead. However, if it works already, then I suppose there's no issue